Dockerfile.jupyter

```diff
@@ -8,7 +8,7 @@ USER root
 RUN apt update && apt install jq -y
 
 # Install libssl1.1 from bullseye
-RUN BULLSEYE_VERSION=$(wget -qO- 'https://sources.debian.org/api/src/openssl/' | jq -r '.versions[] | select(.suites[]=="bullseye") | .version') && \
+RUN BULLSEYE_VERSION=$(wget -qO- 'https://qa.debian.org/madison.php?package=openssl&table=debian&a=&c=&s=bullseye-security&text=on#' | jq -Rn 'input | split(" | ")[1]') && \
```

Comment:
maybe this source (for current openssl version) is more reliable than the previous?